### PR TITLE
[DEVX-956] Add reporters documentation for cypress 

### PR DESCRIPTION
**Pull request by FriggaHel**
**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple 
and short as possible.

### Description

Add references to cypress `reporters` field.

Should not be merged before the new release au `sauce-cypress-runner` and `saucectl` is made !

### Motivation and Context

New feature.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/master/CONTRIBUTING.MD) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 1]
```diff
@@ -408,6 +408,63 @@ For additional information regarding cypress configurations, please consult the
...
+      options:
+        reportDir: cypress/report
+        charts: true
+        reportPageTitle: Cypress running on Sauce
+```
+
+:::note
+This field has the same purpose as `reporterOptions` in `cypress.json` file.
+
+Reporter options are specific for each reporter. Take a look to your reporter options in order to know which options are available.
```
Comment (alexplischke) [inline docs/testrunner-toolkit/configuration/cypress.md:463]:
```suggestion
Reporter options are specific for each reporter. Consult your reporter documentation, in order to know which options are available.
```

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 2]
```diff
@@ -408,6 +408,63 @@ For additional information regarding cypress configurations, please consult the
...
+Reporters functionality is provided through [cypress-multi-reporter](https://www.npmjs.com/package/cypress-multi-reporters).
+In order for your additional reporter to work, it must be compatible with that plugin.
+:::
+
+---
+
+#### `name`
+<p><small>| REQUIRED | STRING |</small></p>
+
+The name of the reporter to enable.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress.md:435]:
```suggestion
The name of the reporter to enable, which corresponds to the `reporter` property in the `cypres.json` file.
```

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 3]
```diff
@@ -408,6 +408,63 @@ For additional information regarding cypress configurations, please consult the
...
+
+```yaml
+      name: cypress-mochawesome
+```
+
+:::note
+This field has the same purpose as `reporter` in `cypress.json` file.
+
+Some reporters may require you to install dependencies.
+:::
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress.md:445]:
```suggestion
:::note
Some reporters may require you to install dependencies.
:::
```

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 4]
```diff
@@ -408,6 +408,63 @@ For additional information regarding cypress configurations, please consult the
...
+
+---
+
+#### `options`
+<p><small>| OPTIONAL | OBJECT |</small></p>
+
+The options of reporters you want to add.
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress.md:452]:
```suggestion
Any relevant settings that are be supported by the specified reporter. These properties correspond to the `reporterOptions` object in the `cypress.json` file.
```

File: docs/testrunner-toolkit/configuration/cypress.md [hunk 5]
```diff
@@ -408,6 +408,63 @@ For additional information regarding cypress configurations, please consult the
...
+        reportDir: cypress/report
+        charts: true
+        reportPageTitle: Cypress running on Sauce
+```
+
+:::note
+This field has the same purpose as `reporterOptions` in `cypress.json` file.
+
+Reporter options are specific for each reporter. Consult your reporter documentation, in order to know which options are available.
+:::
```
Comment (sweeneyskirt-sl) [inline docs/testrunner-toolkit/configuration/cypress.md:464]:
```suggestion
```

**Review by sweeneyskirt-sl — CHANGES_REQUESTED** [review]

**Comment by sweeneyskirt-sl** [comment]
Hey @alexplischke -- I made a couple edits. Hoping you will give a second scan before we merge. 🙏

**Pull request merged**